propka/lib.py

```diff
@@ -272,3 +279,16 @@ def writeFile(filename, lines):
     f.close()
 
 
+no_print = False                # module-scope. Set when options are parsed.
+def info(*args, **kargs):
+    """Behaves like print(), unless the --no-print option is set. """
+    global no_print
+    if not no_print:
+        print(*args, **kargs)
+
+
+def warn(*args, **kargs):
+    """Behaves like print() - possibly to be replaced by a logger"""
```

Comment:
I think we should just go ahead and do this now.  Instead of a global "no_print" we could have a global "logger" variable.   If options.no_print is set (or options.verbosity is "none"), we would set the logger level to CRITICAL.  If not, it would be set to INFO.

Then we can replace these 2 functions with:

def info(*args):
    logging.info(' '.join(map(str, args)))

def warn(*args):
    logging.warning(' '.join(map(str, args)))